### PR TITLE
Boston Housing Price Demo added under examples folder

### PR DESCRIPTION
**Pull request by msulemannkhan**
## Description
This is an example for PyDP using Boston housing price dataset

## Affected Dependencies
No

## How has this been tested?
Manually verified

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [x] My changes are covered by tests

### CODE REVIEW

**Review by chinmayshah99 — CHANGES_REQUESTED** [review]
Can we not add dataset from sklearn and directly call it from source?
or maybe call from a URL?
Calling it from sklearn means sklearn needs to be present

**Comment by msulemannkhan** [comment]
Right, I have added the data file.

**Review by Frank-7 — CHANGES_REQUESTED** [review]
If we are going to read the dataset locally there is no need for this line on the second cell:
from sklearn.datasets import load_boston

Please remove that line and I will approve the change

**Comment by msulemannkhan** [comment]
Right, I have updated removed this line.

**Review by Frank-7 — APPROVED** [review]
Thanks, that would do it
